packages/dashboard/src/customization/widgets/table/component.tsx

```diff
@@ -55,7 +59,13 @@ const TableWidgetComponent: React.FC<TableWidget> = (widget) => {
   };
 
   return (
-    <div onMouseDown={handleMouseDown}>
+    <div
+      onMouseDown={handleMouseDown}
+      style={{
+        maxHeight: chartSize.height > 520 ? '520px' : chartSize.height,
```

Comment:
please create constants for value 520 and 200 with appropriate name

Reply:
done